nina_xmpp.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=XMPP-Bot for NINA warnings
+After=ejabberd.service
```

Comment:
I don't have ejabberd on my system :)

Reply:
In that case
`After=<your_server_unit_here>`
would be the right choice for you =)

Or we could do a general
`After=network.target`
but then there would be the chance that the server isn't available while the bot tries to connect.

Reply:
> I don't have ejabberd on my system :)

That shouldn't be a problem, since `After` doesn't imply `Requires` (or even `Wants`, for that matter). In fact, I'd vote to include all XMPP server units that come to mind (prosody.service in my case). If the bot is run on the same system as the XMPP server, this makes sense. If not, it does no harm.

OTOH, I'm torn on the question how the bot should behave when connecting fails on startup. Currently, I'm relying on systemd to restart the whole process (thanks to `Restart=always`)